### ISSUE TITLE
Error: ENOENT: no such file or directory, stat daterangepicker.js

### ISSUE THREAD

**Issue by ngelx** [josemarluedke/ember-cli-daterangepicker#48]
I could not install and get this addon to work. After ` ember install ember-cli-daterangepicker`, when I try to run `ember serve`:

```
Error: ENOENT: no such file or directory, stat '/vagrant/ember-app/tmp/source_map_concat-input_base_path-m010cHhG.tmp/0/bower_components/bootstrap-daterangepicker/daterangepicker.js'
....
```

* ember-cli: 2.4.3
* node: 5.10.0

Thanks in advance

**Comment by marclundgren** [comment]
@ngelx I have the same problem. Did you find a solution for this?

**Comment by josemarluedke** [comment]
Just released version 0.1.0 that moves dependencies to npm. It should fix most of the problems related to the dependencies. Closing this for new. If you still have it, let me know.
